_content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "A Hoard of Inscribed Gandharan Metalware"
+authors:
+  - "Richard Salomon"
+external_url: "https://poj.peeters-leuven.be/content.php?id=3291581&url=article"
```

Comment:
Maybe save our users one click by using this instead: https://poj.peeters-leuven.be/content.php?id=3291581&url=article&download=yes